yt_dlp/postprocessor/modify_chapters.py

```diff
@@ -34,23 +33,13 @@ def run(self, info):
         if not chapters:
             chapters = [{'start_time': 0, 'end_time': duration, 'title': info['title']}]
 
-        # TODO: Refactor _remove_marked_arrange_sponsors to not need this
-        for c in sponsor_chapters:
-            c['categories'] = [(c['category'], c['start_time'], c['end_time'])]
-
```

Comment:
Moving this down breaks the code. Try running `yt-dlp fFFSKedy9f4 --sponsorblock-mark all`

Reply:
Moved this part to `SponsorBlockPP`.